uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Mail/MessageTest.php

```diff
@@ -37,13 +38,14 @@ public function testGetRecipientEmailsReturnsCorrectlyWhenLeaveContactHasLeaveAp
     $manager2 = ContactFabricator::fabricateWithEmail([
       'first_name' => 'Manager2', 'last_name' => 'Manager2'], 'manager2@dummysite.com'
     );
```

Comment:
Now that we're mocking the LeaveManager service to return the managers, do we still need to fabricate them using the Contact Fabricator?

Reply:
Yes because, there's a call to get the Email of the contacts in the Message class